### ISSUE TITLE
Unable to get correct data from ECU

### ISSUE THREAD

**Issue by arthur2017-ui** [ksheumaker/homeassistant-apsystems_ecur#114]
Every week during 1 or 3 days, the ECU integration doesn't show the measures  of Power and Energy ; message on dashborad  is ECU Power/Energy "Not available". Integration says "Unable to get correct data from ECU, and no cached data. See log for details, and try power cycling the ECU."  
After ECU-C power cycling and reconfig IP address, I could restart and get fresh data.... but only for 4 to 6 days per week.

Is this a known issue ?
Thanks in advance

**Comment by HAEdwin** [comment]
Yes it is, despite efforts to solve this it seems more firmware related and the automation should be used to solve the issue. Read the instructions on how to do this in de readme.
